.flake8

```diff
@@ -5,6 +5,8 @@ ignore =
 select =
     C4,
     E722,
+    E711,
+    E712,
```

Comment:
As discussed, let's not add this rule. `== True` is perfectly valid when working with symbolic booleans. See the docstring of `S.true`.

Reply:
Also, this is unrelated to this PR, but can we add comments to this file explaining what each rule is, like in pyproject.toml.